src/cbmc/cbmc_parse_options.cpp

```diff
@@ -430,6 +460,50 @@ void cbmc_parse_optionst::get_command_line_options(optionst &options)
 
 /*******************************************************************\
 
+Function: cbmc_parse_optionst::options_exclusive
+
+  Inputs:
+
+ Outputs:
+
+ Purpose: check conflicts between options: not allowed together
+
+\*******************************************************************/
+
+bool cbmc_parse_optionst::options_exclusive(const char *opt1, const char *opt2)
+{
+  if(cmdline.isset(opt1) && cmdline.isset(opt2))
+  {
+    error() << "--" << opt1 << " cannot be used with --" << opt2 << eom;
+    return true;
+  }
+  return false;
+}
+
+/*******************************************************************\
+
+Function: cbmc_parse_optionst::options_inclusive
+
+  Inputs:
+
+ Outputs:
+
+ Purpose: check conflicts between options: opt1 only if opt2
+
+\*******************************************************************/
+
+bool cbmc_parse_optionst::options_inclusive(const char *opt1, const char *opt2)
+{
+  if(cmdline.isset(opt1) && !cmdline.isset(opt2))
+  {
+    error() << "--" << opt1 << " can only be used with --" << opt2 << eom;
+    return true;
+  }
+  return false;
+}
+
+/*******************************************************************\
+
```

Comment:
Please make separate pull request for the two above; this seems unrelated to incremental.